assets/src/dashboard/app/views/shared/pageHeading.js

```diff
@@ -36,7 +36,7 @@ const Container = styled.div`
   display: flex;
   justify-content: space-between;
   flex-wrap: wrap;
-  margin: ${({ theme }) => `40px ${theme.pageGutter.small.desktop}px`};
+  margin: ${({ theme }) => `0 ${theme.pageGutter.small.desktop}px 40px`};
```

Comment:
I updated it so the header pushes down instead of the content below it pushing up.  This way, we can stack things on top of each other and they should all space out appropriately as long as everything pushes down (i.e. margin-bottom) as needed.